src/main/resources/reference.conf

```diff
@@ -201,10 +237,10 @@ cassandra-query-journal {
   
   # How many events to fetch in one query (replay) and keep buffered until they
   # are delivered downstreams.
-  max-buffer-size = 100
+  max-buffer-size = 10000
```

Comment:
On the other hand small buffer means we need to fetch from database more frequently which may increase latency. I will decrease to the original value.